src/Controller/BrowserAgentController.php

```diff
@@ -58,25 +58,16 @@ public function generate()
 
         try {
             $result = $builder->build($_POST, $this->getDoctrine());
-            $response = new Response(
-                json_encode([
-                    'error' => '',
-                    'build_id' => $result,
-                ]
-                )
-            );
+
+            return $this->json([
```

Comment:
Do you think that instead of having return here we may create an array here and later at the end of this controller action we can call $this->json()?

We can have an array with the same name in the `catch` logic as well.

Reply:
Hello @ceckoslab 
Fixed. Now it returns $response array built within try/catch